static-site/variables.tf

```diff
@@ -0,0 +1,32 @@
+# aws region
+provider "aws" {
+  region = "us-east-1"
+}
+
+// root domain name
+variable "root_domain_name" {
+  default = "digital.mass.gov"
+}
+
+// new statis sites are subdomains under *.digital.mass.gov
+variable "sub_domain_name" {
```

Comment:
Can we rename this variable to `domain_name`?  This name would be very confusing if we ever wanted to create a static site that isn't a subdomain.